Answers.md

```diff
@@ -0,0 +1,17 @@
+## Task 2: Responses
+Answer the following questions to the best of your ability. You can exercise your Googling skills and use training kit.  Open up the [Answers.md](Answers.md) file and record your responses there.
```

Comment:
Really great answers here Mehdi 👍 
Answering questions this thoroughly will pay big dividends for you when in it comes time to interview.